generation/3d_ddpm/3d_ddpm_tutorial.ipynb

```diff
@@ -0,0 +1,770 @@
+{
```

Comment:
<p>I don't know why it's complaining about torch.load here, but we could just remove this Warning message? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1874/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>